src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -1141,11 +1168,11 @@ internal override bool CanAddProjectReference(ProjectId referencingProject, Proj
         /// A trivial implementation of <see cref="IVisualStudioWorkspaceHost" /> that just
         /// forwards the calls down to the underlying Workspace.
         /// </summary>
-        protected class VisualStudioWorkspaceHost : IVisualStudioWorkspaceHost, IVisualStudioWorkingFolder
+        private sealed class VisualStudioWorkspaceHost : IVisualStudioWorkspaceHost, IVisualStudioWorkingFolder
```

Comment:
is this safe?  @jasonmalinowski does anyone use this host?  I don't believe TS does.  But i'm not sure about codelens and others.